lib/ansible/module_utils/urls.py

```diff
@@ -814,6 +824,7 @@ def fetch_url(module, url, data=None, headers=None, method=None,
     Fetches a file from an HTTP/FTP server using urllib2.  Requires the module environment
     '''
 
+    #FIXME: not really working here as class defs fail when using it for their inheritance
```

Comment:
urllib2 comes from the stdlib so my recommendation is to remove this check and the HAS_URLLIB2 attribute.